go_templates/query.vm

```diff
@@ -56,7 +56,11 @@ type ${paramsType} struct {
 #if ( "#queryParamTypeName($qp)" != "" )
 
   // $str.formatDoc("#queryParamTypeName($qp) ${str.uncapitalize($qp.doc)}", "  // ")
-  #queryParamTypeName($qp) #oapiToGo($qp, false) `url:"$qp.propertyName,omitempty"`
+  #if ( $qp.arrayType == "string" )
+    #queryParamTypeName($qp) []string `url:"$qp.propertyName,omitempty,comma"`
```

Comment:
What does `comma` mean here?

Reply:
It means this part of the url will be like `param=ab,cde` instead of `param=ab&param=cde`.